### ISSUE TITLE
[Feature] Web UI for Alarm/Alert management

### ISSUE THREAD

**Issue by gamefundas** [netdata/netdata#6330]
It would be good to control all aspects of alerting/suppression from the UI, as configuring alerts/alarms through configuration file is not very practical. Over time the no of nodes increases and it gets harder to manage alerts.

Few aspects of the configuration such as email/webhook/pagerduty can global. Suppression and sliding window mechanisms to control the amount of alerts raised on the service level provides a better means to reduce noise in incident management systems.

**Comment by cakrit** [comment]
cc @gpapamathND

@gamefundas you're absolutely correct and it's one of the big convenience features we have on the netdata cloud roadmap, especially wrt the batch deployment of such configurations, or central health monitoring. The first idea is to have a method of applying configs to all agents, the second is to have the cloud collect and correlate alarms, issue notifications. Both are valid use cases.

In addition, for the single node UI, we were considering adding configuration wizards (see #5125) . Of course their output could be added to  a configuration management tool so that other nodes can get the same settings as well. What we don't intend to do is convert one node (e.g. a master) into some kind of management node for the entire insfrastructure. We want the cloud to take that responsibility. 

There's definitely a lot of work to be done in this area and we'd appreciate any hep we can get.

**Comment by cakrit** [comment]
Blocked by #6432

**Comment by cakrit** [comment]
We won't track this here any more, treating it as duplicate of #6432
